spectre.py

```diff
@@ -1,255 +1,352 @@
+    "Gamma":  np.array((255, 255, 255),'f')/255.,
+    "Gamma1": np.array(( 97,  57,  21),'f')/255.,
+    "Gamma2": np.array(( 64,  64,  64),'f')/255.,
+    "Delta":  np.array((  2, 129,  33),'f')/255.,
+    "Theta":  np.array((  0,  76, 255),'f')/255.,
+    "Lambda": np.array((118,   0, 136),'f')/255.,
+    "Xi":     np.array((229,   0,   0),'f')/255.,
+    "Pi":     np.array((255, 175, 199),'f')/255.,
+    "Sigma":  np.array((115, 215, 238),'f')/255.,
+    "Phi":    np.array((255, 141,   0),'f')/255.,
+    "Psi":    np.array((255, 238,   0),'f')/255.
+}
+## draw Polygons Svg by drawsvg #####
+import drawsvg
 
 start = time()
-d.save_svg("spectre.svg")
-time3 = time()-start
-print(f"SVG drawing took {round(time3, 4)} seconds")
-print(f"total processing time {round(time1+time2+time3, 4)} seconds, {round(1000000*(time1+time2+time3)/num_tiles, 4)} μs/tile")
\ No newline at end of file
+
+def flattenPts(lst): # drowsvg
+    return [item for sublist in lst for item in sublist] # drowsvg
+
+SPECTRE_SHAPE = drawsvg.Lines(*flattenPts([p for p in SPECTRE_POINTS]), stroke="black", stroke_width=0.5,close=True) # drowsvg
+Mystic_SPECTRE_SHAPE = drawsvg.Lines(*flattenPts([p for p in Mystic_SPECTRE_POINTS]), stroke="black",   stroke_width=0.5, close=True) # drowsvg
+
+svgContens = drawsvg.Drawing(transformation_max - transformation_min,
+                    transformation_max - transformation_min,
+                     origin="center") # @TODO: ajust to polygons X-Y min and max. 
+num_tiles = 0 # drowswvg
+
+def drawPolygon2Svg(T, label): #drowsvg
+    """
+    T: transformation matrix
+    label: label of shape type
+    """
+    global num_tiles,svgContens
+    num_tiles += 1
+
+    fill = f"rgb({int(round(COLOR_MAP[label][0]* 255, 0))}, {int(round(COLOR_MAP[label][1]* 255,0))}, {int(round(COLOR_MAP[label][2]* 255,0))})"
+    stroke_f = "gray" # tile stroke color
+    stroke_w = 0.1 if (fill[0] != 0) | (fill[1] != 0) | (fill[2] != 0) else 0 # tile stroke width
+    shape = SPECTRE_SHAPE if label != "Gamma2" else Mystic_SPECTRE_SHAPE  # geometric points used.
+    # print(f"transform-matrix,{T[0,0]},{T[1,0]},{T[0,1]},{T[1,1]},{T[0,2]},{T[1,2]}")
+
+    svgContens.append(drawsvg.Use(
+        shape,
+        0, 0,
+        transform=f"matrix({T[0,0]} {T[1,0]} {T[0,1]} {T[1,1]} {T[0,2]} {T[1,2]})",
+        fill=fill,
+        stroke=stroke_f,
+        stroke_width=stroke_w))
+
+tiles["Delta"].drawPolygon(drawPolygon2Svg) # updates num_tiles
+saveFileName = f"spectre_tile{Edge_a:.1f}-{Edge_b:.1f}_{N_ITERATIONS}-{num_tiles}useRef.svg"
+svgContens.save_svg(saveFileName)
+time4 = time()-start
+print(f"drowsvg: SVG drawing took {round(time4, 4)} seconds, generated {num_tiles} tiles")
+print("drowsvg: drawPolygon save to " + saveFileName)
+print(f"drowsvg: total processing time {round(time1+time4, 4)} seconds, {round(1000000*(time1+time4)/num_tiles, 4)} μs/tile")
+
+# exit(0)
+
+# draw Polygons Svg by matplotlib #####
+# import matplotlib.pyplot as plt
```

Comment:
Instead of having this commented out code, could you make a separate file for drawing the tiles with matplotlib (and maybe similar for drawsvg as well), importing the necessary tile generating functions from the main file? That would be much nicer.